charts/openfaas/v5.7.2/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+description: Enable Kubernetes as a backend for OpenFaaS (Functions as a Service)
+name: openfaas
+version: 5.7.2
+sources:
+- https://github.com/openfaas/faas
+- https://github.com/openfaas/faas-netes
+home: https://www.openfaas.com
+icon: https://raw.githubusercontent.com/openfaas/media/master/OpenFaaS_logo_stacked_opaque.png
```

Comment:
@guangbochen Is a link to an icon like this okay, or should I copy them into the chart folder so we can reference them locally?

Reply:
Reference them locally @PennyScissors as an air gapped setup wouldn't be able to grab that image.

Reply:
Done